### ISSUE TITLE
Wrong element focused when opening the conversation settings

### ISSUE THREAD

**Issue by danxuliu** [nextcloud/spreed#6097]
When the conversation settings are opened [the link share settings are focused](https://github.com/nextcloud/spreed/blob/43834cb456767827db6c0817258d8e20102c2076/src/components/ConversationSettings/ConversationSettingsDialog.vue#L189). However, if the current user does not have full moderation permissions the link share settings do not exist, so focusing the element fails. Moreover, even if the current user has full moderation permissions the element focused is not the first one in the dialog, so tabbing after the dialog is opened moves to an unexpected element.

In any case focusing an element in the dialog once the dialog is opened might be something to address in the nextcloud-vue library instead, for example by focusing the first element in the navigation list one the application settings dialog is opened.

## Steps to reproduce

- Open a conversation as a regular participant (not a moderator)
- Show the console in the developer tools of the browser
- Open the conversation dialog

### Expected behaviour

No error is shown in the browser console.

### Actual behaviour

`TypeError: this.$refs.linkShareSettings is undefined` error is shown in the browser console.

**Comment by nickvergessen** [comment]
Solved by now
